pkg/docker/client.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2021 Terminus, Inc.
+
```

Comment:
The `docker-client` provider should be provided by erda-infra

Reply:
done